_posts/2024-11-12-test-markdown.md

```
@@ -0,0 +1,64 @@
+```shell
+go test -coverprofile=.coverage.out `go list ./...` -short
+# github.com/shirou/gopsutil/cpu
+../../go/pkg/mod/github.com/shirou/gopsutil@v2.20.9+incompatible/cpu/cpu_darwin_cgo.go:13:5: error: 'TARGET_OS_MAC' is not defined, evaluates to 0 [-Werror,-Wundef-prefix=TARGET_OS_]
+   13 | #if TARGET_OS_MAC
+      |     ^
+1 error generated.
+
+```
+
+详细的报错具体大概是这种，如果想了解错误的原因可以查看官方Issue或者文档github.com/shirou/gopsutil包版本太低导致的TARGET_OS_MAC错误 
+https://github.com/shirou/gopsutil/issues/976
```

Markdownlint

Bare URL used

(MD034, no-bare-urls)